phpstan.neon

```diff
@@ -51,21 +47,11 @@ services:
 	# 	tags:
 	# 		- phpstan.rules.rule
 
-	- # https://github.com/symplify/phpstan-rules/blob/main/docs/rules_overview.md#noissetonobjectrule
-		class: Symplify\PHPStanRules\Rules\NoIssetOnObjectRule
```

Comment:
not sure when they removed it but I donot find the rule here https://github.com/symplify/phpstan-rules/tree/main/src/Rules